<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContact</class>
 <widget class="QDialog" name="AddContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/修改联系人</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDateEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>住址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>生日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>关系</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>家人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爱人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>朋友</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>同学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>同事</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老师</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="homeEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
